src/views/default-view.ts

```diff
@@ -389,6 +388,17 @@ export class DefaultView<N extends INodeBase, E extends IEdgeBase> implements IO
     }
   };
 
+  private _initCanvas(): HTMLCanvasElement {
+    const canvas = document.createElement('canvas');
+    canvas.id = 'orbCanvas';
```

Comment:
What will happen if there is a `HTMLElement` with the same `id` in the application using Orb? Do we want to introduce `uuid` for these kind of ids?

Reply:
Good comment, I was just wondering that. I'm assigning `uuid`s in `ngx-orb`, but we may want to have those in here as well. But I'm not sure wether this is the responsibility of the user or the library. It also makes sense to me that the user takes care of his own container and orb instances.

This is effectively a leftover so I'll delete it. I used this to get the canvas element but then i realized that I already have a reference to the element (`this._canvas`)